Networking/Networking/Remote/BlazeRemote.swift

```diff
@@ -9,6 +9,31 @@ public protocol BlazeRemoteProtocol {
     ///    - pageNumber: the page number of campaign to load.
     ///
     func loadCampaigns(for siteID: Int64, pageNumber: Int) async throws -> [BlazeCampaign]
+
+    /// Fetches target languages for campaign creation.
+    /// - Parameters:
+    ///    - siteID: WPCom ID for the site to create ads campaigns.
+    ///
+    func fetchTargetLanguages(for siteID: Int64) async throws -> [BlazeTargetLanguage]
+
+    /// Fetches target devices for campaign creation.
+    /// - Parameters:
+    ///    - siteID: WPCom ID for the site to create ads campaigns.
+    ///
+    func fetchTargetDevices(for siteID: Int64) async throws -> [BlazeTargetDevice]
+
+    /// Fetches target topics for campaign creation.
+    /// - Parameters:
+    ///    - siteID: WPCom ID for the site to create ads campaigns.
+    ///
+    func fetchTargetTopics(for siteID: Int64) async throws -> [BlazeTargetTopic]
+
+    /// Fetches target locations for campaign creation.
+    /// - Parameters:
+    ///    - siteID: WPCom ID for the site to create ads campaigns.
+    ///    - query: Keyword to search for locations.
```

Comment:
```suggestion
    ///    - query: Keyword to search for locations. Requires a minimum of 3 characters, or else will return error.
```

Add a bit of usage guide.